acai-ios/CollectionViewCells/CartCollectionViewCell.swift

```diff
@@ -0,0 +1,195 @@
+//
+//  CartCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Jaewon Sim on 4/5/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+protocol CartCollectionViewCellDelegate: class {
+    func valueIncremented()
+    func valueDecremented()
+}
+
+class CartCollectionViewCell: UICollectionViewCell {
+
+    // MARK: View vars
+    var imageView: UIImageView!
+    var titleLabel: UILabel!
+    private var ingredientsCollectionView: UICollectionView!
+    private var decrementButton: UIButton!
+    private var currentCountLabel: UILabel!
+    private var incrementButton: UIButton!
+    private var totalPriceLabel: UILabel!
+    private var line: UIView!
+
+    let collectionCellHeight = 24
+
+    // MARK: Data
+    private var ingredients: [OrderOption] = []
+    let ingredientsCellReuseIdentifier = "ingredientsCellReuseIdentifier"
+
+    // MARK: Delegate
+    weak var delegate: CartCollectionViewCellDelegate?
```

Comment:
same here. can you make a github issue to add `private` to variables and functions where possible

Reply:
added issue